vite.config.ts

```diff
@@ -1,18 +1,30 @@
+/// <reference types="vitest" />
 import path from 'path';
 import { UserConfig } from 'vite';
-import svelte from '@sveltejs/vite-plugin-svelte';
+import { svelte } from '@sveltejs/vite-plugin-svelte';
```

Comment:
By updating `@sveltejs/vite-plugin-svelte` to `1.0.0-next.36` the library moved to named exports.